x-pack/plugins/security_solution/server/lib/detection_engine/signals/executors/eql.ts

```diff
@@ -76,12 +77,21 @@ export const eqlExecutor = async ({
       index: ruleParams.index,
     });
 
+    const esFilter = getQueryFilter(
+      ruleParams.query,
+      ruleParams.language,
+      ruleParams.filters || [],
+      inputIndex,
+      exceptionItems
+    );
+
```

Comment:
`exceptionItems` are processed by both this call and `buildEqlSearchRequest`, resulting in the exceptions being included in the query twice. Let's pass `ruleParams.filters` in to `buildEqlSearchRequest` directly instead to keep more of the logic contained in the unit-testable `buildEqlSearchRequest`. Let's also update the unit tests to test this `Filter` processing logic.

I don't think we want to pass the EQL query in to `getQueryFilter` either, since the EQL query is used in the string form in the generated search request - it doesn't need any additional processing. We can probably pass an empty array to this function to process only the filters.